packages/graphql-hooks/index.d.ts

```diff
@@ -122,7 +123,8 @@ export const ClientContext: React.Context<GraphQLClient>
 type ResetFunction = (desiredState?: object) => void
 
 interface ClientOptions {
-  url: string
+  url?: string
+  local?: LocalQueries
```

Comment:
this is no longer needed if we're extending the client, right?

Reply:
I still need to update those definitions with the new scheme, I think this bit was originally added by Guido with the previous implementation of the local client

Reply:
It's good to have this comment as a reminder to update the types.

It's something easy to forget :)

Reply:
let's add it as an issue, but clearly not to be addressed in this same PR

Reply:
Just noticed there's already one created recently #796